<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaviWidget</class>
 <widget class="NaviWidget" name="NaviWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NaviWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>800</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="edit_search">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>251</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>431</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>选择起点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>选择终点</string>
    </property>
   </widget>
   <widget class="PainterWidget" name="mapWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>40</y>
      <width>980</width>
      <height>750</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enlargeBtn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>放大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reduceBtn">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>enlargeBtn</zorder>
   <zorder>reduceBtn</zorder>
   <zorder>edit_search</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PainterWidget</class>
   <extends>QWidget</extends>
   <header>painterwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NaviWidget</class>
   <extends>QWidget</extends>
   <header>naviwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
